logical/error.go

```diff
@@ -5,21 +5,24 @@ type HTTPCodedError interface {
 	Code() int
 }
 
-func CodedError(c int, s string) HTTPCodedError {
-	return &codedError{s, c}
+func CodedError(status int, msg string) HTTPCodedError {
+	return &codedError{
+		Status:  status,
+		Message: msg,
+	}
 }
 
 type codedError struct {
-	s    string
-	code int
+	Status  int
```

Comment:
This is one of the reasons I opted to just use a struct instead of the interface in GH-3443. The struct fields have to be public to be encoded/decoded by gob.